### ISSUE TITLE
Motorcycle plates detection to Brazil

### ISSUE THREAD

**Issue by toniesteves** [openalpr/openalpr#748]
Hi! I can't perform motorcycle plates detection to Brazilian plates. Have anyone already done it?

OSX High Sierra
MacBook Pro (13-inch, Mid 2012)
Processor 2.5 GHz Intel Core i5

**Comment by usantos** [comment]
Do you detect car plate brasil? What did you use to set it up?

**Comment by toniesteves** [comment]
Yes I detect Brazil car plates, sometimes not precisely, but works. For it I use a piece of code in OpenALPR doc:

https://github.com/openalpr/openalpr/tree/master/src/bindings/python <https://github.com/openalpr/openalpr/tree/master/src/bindings/python>

Toni Esteves
Bacharel em Sistemas de Informação
Especialista em Administração de Bancos de Dados
Twitter: @toni_esteves
Skype: toni.esteves
https://github.com/toniesteves/CV <https://github.com/toniesteves/CV>
http://lattes.cnpq.br/7013943111832654 <http://lattes.cnpq.br/7013943111832654>
http://www.linkedin.com/in/toniesteves <http://www.linkedin.com/in/toniesteves>

**Comment by xtianhb** [comment]
Hello,
Have you tried using --debug option? To gather more info?
Are you using standard configuration?
Images size?

**Comment by SouzaRDG** [comment]
Helo,
have you solved your problem? I'm having issues with motorcycles too.
Tried some modifications on br2.config, but with no luck...

heres the some of the images I've tried:
![m01](https://user-images.githubusercontent.com/33104876/58375461-daa2cd80-7f29-11e9-9c7c-569e86f13f7b.jpg)
![m02](https://user-images.githubusercontent.com/33104876/58375463-e68e8f80-7f29-11e9-831d-2130140ec7b3.jpg)

**Comment by rafael-pereira-atman** [comment]
> Helo,
> have you solved your problem? I'm having issues with motorcycles too.
> Tried some modifications on br2.config, but with no luck...
> 
> heres the some of the images I've tried:
> ![m01](https://user-images.githubusercontent.com/33104876/58375461-daa2cd80-7f29-11e9-9c7c-569e86f13f7b.jpg)
> ![m02](https://user-images.githubusercontent.com/33104876/58375463-e68e8f80-7f29-11e9-831d-2130140ec7b3.jpg)

Hi, I've tested your images and it worked. I've changed the prewarp configurations for each image. As follow the configuration.

Image above
prewarp = planar,620.000000,350.000000,-0.000350,0.000150,-0.000000,1.015000,0.750000,0.000000,0.000000

Image below
prewarp = planar,605.000000,340.000000,-0.000000,0.000000,-0.000000,1.000000,0.740000,0.000000,0.000000
